ui/plugin-system/src/runtime/time-range.ts

```diff
@@ -12,11 +12,13 @@
 // limitations under the License.
 
 import { createContext, useContext } from 'react';
-import { AbsoluteTimeRange, DurationString } from '@perses-dev/core';
+import { AbsoluteTimeRange, TimeRangeValue } from '@perses-dev/core';
 
 export interface TimeRange {
-  timeRange: AbsoluteTimeRange;
-  defaultDuration: DurationString;
+  timeRange: TimeRangeValue;
+  absoluteTimeRange: AbsoluteTimeRange;
```

Comment:
If a plugin is only concerned with the resolved timestamps (the date object/unix epoch ms), do they use `absoluteTimeRange`?

Reply:
yeah that's right, maybe I should rename or potentially consolidate these 🤔  (I also just added previousTimeRange in my [unzoom PR](https://github.com/perses/perses/pull/537))